x/auth/ante/sigverify.go

```diff
@@ -94,6 +96,29 @@ func (spkd SetPubKeyDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simulate b
 		spkd.ak.SetAccount(ctx, acc)
 	}
 
+	// Also emit the following events, so that txs can be indexed by these
+	// indices:
+	// - signature (via `tx.signature='<sig_as_base64>'`),
+	// - address and sequence (via `message.sender=<addr> AND tx.sequence='<seq>' `).
+	sigs, err := sigTx.GetSignaturesV2()
+	if err != nil {
+		return ctx, err
+	}
+
+	var events sdk.Events
+	for _, sig := range sigs {
+		sigBz, err := signatureDataToBz(sig.Data)
```

Comment:
We should probably emit a signature for each individual single signer in a multisig as that is the only data an individual signer actually controls.

Reply:
Makes sense. Should we also emit the aggregated final signature?

Reply:
Added in 76f6fdd880f9579454bc6df4f48a15e3b28daff7